README.md

```diff
@@ -1,12 +1,43 @@
+
 [![Join the chat at https://gitter.im/fetch-amul-topicals/Lobby](https://badges.gitter.im/fetch-amul-topicals/Lobby.svg)](https://gitter.im/fetch-amul-topicals/Lobby?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
-A Python script that fetches details all ‘topicals’ across all years from the Amul website (http://www.amul.com/m/amul-hits) and has the feature to download the pictures by year into your local system
+It uses a Python script (aka **web scraper**) that fetches details (here ‘topicals’) across all years from the Amul's website (http://www.amul.com/m/amul-hits) and has the feature to download the pictures chronologically into your local system.
```

Comment:
It is a Python script in itself. This is an unnecessary expansion of the already existing description. Chuck preferably.